tests/cpp/aot/aot_save_load_test.cpp

```diff
@@ -311,11 +311,10 @@ TEST(AotSaveLoad, VulkanNdarray) {
   auto g = std::make_unique<Graph>("test");
   auto seq = g->seq();
   auto arr_arg = aot::Arg{
-      "arr", PrimitiveType::i32.to_string(), aot::ArgKind::NDARRAY, {}};
+      aot::ArgKind::NDARRAY, "arr", PrimitiveType::i32.to_string(), {}};
```

Comment:
Google's enum style is `kOption`.. https://google.github.io/styleguide/cppguide.html#Enumerator_Names